manifests/osfamily/debian.pp

```diff
@@ -5,117 +5,124 @@
 
     include ::apt
 
-    if getvar('::puppet_agent::is_pe') == true {
-      $pe_server_version = pe_build_version()
-      if $::puppet_agent::source {
-        $source = $::puppet_agent::source
+    if $::puppet_agent::absolute_source {
```

Comment:
Seems like this `if` needs to move outside the `if getvar('::puppet_agent::manage_repo') == true` on line 4.  (Should be similar to the suse.pp & redhat.pp manifests.)

Reply:
Oh, whoops :) Didn't notice the if on line for.